src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ConvictionTransformer.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.ActiveOffence
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.community.Conviction
+
+@Component
+class ConvictionTransformer {
+  fun transformToApi(conviction: Conviction) = conviction.offences?.map {
+    ActiveOffence(
+      deliusEventNumber = conviction.index,
+      offenceDescription = "${it.detail.mainCategoryDescription} - ${it.detail.subCategoryDescription}",
```

Comment:
👍 this makes sense as sometimes (though not usually) the sub category description provides additional detail which may be important. Perhaps a "nice to have" would be a function which only includes the sub-category description if it is different from the main-category description...

Reply:
Good shout - have added a check to see if they're the same